### PR TITLE
v7.18.0

### PR DESCRIPTION
**Pull request by KenanYusuf**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,81 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.18.0
+
+_Sep 19, 2024_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
```
Comment (KenanYusuf) [inline CHANGELOG.md:11]: Any suggestions for highlights?

Comment (flaviendelangle) [reply CHANGELOG.md:11]:
```
- [charts] Add a `PolarProvider` to manage polar axes (#14642) @alexfauquette
```

Maybe, if @alexfauquette thinks it's worth highlighting

**Review by flaviendelangle — APPROVED** [review]
Almost nothing in eXplore this week :laughing:

**Review by LukasTy — COMMENTED** [review]
Could you re-run changelog generation after https://github.com/mui/mui-x/pull/14673 is merged?
We'll have it and https://github.com/mui/mui-x/pull/11780 to include. 👍

File: packages/x-charts-vendor/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts-vendor",
-  "version": "7.16.0",
+  "version": "7.17.0",
```
Comment (KenanYusuf) [inline packages/x-charts-vendor/package.json:3]: @mui/x 

What do we do in cases where a package version is out of sync with others?

Others are being bumped to `7.18.0`, but this is still on `7.16.0`.

File: packages/x-charts-pro/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts-pro",
-  "version": "7.0.0-beta.1",
+  "version": "7.0.0-beta.2",
```
Comment (KenanYusuf) [inline packages/x-charts-pro/package.json:3]: Is this correct? I'm not clear on how prereleases should be bumped.

File: packages/x-charts-vendor/package.json (hunk 2 above)
Comment (LukasTy) [reply packages/x-charts-vendor/package.json:3]: It should be synced.
This means that on last release there was no need to release it, so we skipped it, but whenever we are releasing—we use the version of the release.

File: packages/x-charts-pro/package.json (hunk 3 above)
Comment (LukasTy) [reply packages/x-charts-pro/package.json:3]: Yes, this is correct. 👍

**Comment by cherniavskii** [comment]
@KenanYusuf The failing unit test is flaky, you can ignore it. I believe it will be fixed by https://github.com/mui/mui-x/pull/14583

**Review by MBilalShafi — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,83 @@
...
+Same changes as in `@mui/x-data-grid@7.18.0`, plus:
+
+- [DataGridPro] Fix `onRowsScrollEnd` being triggered instantly when bottom pinned row is present (#14602) @arminmeh
+- [DataGridPro] Fix header filters rendering issue for `isEmpty` and `isNotEmpty` filter operators (#14493) @k-rajat19
+- [DataGridPro] Fix pinned columns in RTL mode (#14586) @KenanYusuf
+
+#### `@mui/x-data-grid-premium@7.18.0` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@7.18.0`.
```
Comment (LukasTy) [inline CHANGELOG.md:38]:
```suggestion
Same changes as in `@mui/x-data-grid-pro@7.18.0`.

### Date and Time Pickers

#### `@mui/x-date-pickers@7.18.0`

- [pickers] Add option to change the order of displayed years (#11780) @thomasmoon
- [pickers] Support `date-fns` v4 (#14673) @LukasTy

#### `@mui/x-date-pickers-pro@7.18.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')

Same changes as in `@mui/x-date-pickers@7.18.0`.
```

**Review by LukasTy — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,99 @@
...
 
+## 7.18.0
+
+_Sep 20, 2024_
+
+We'd like to offer a big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Support [Row spanning](https://mui.com/x/react-data-grid/row-spanning/) on the Data Grid that automatically merges the consecutive cells in a column based on the cell value
+
+  <img width="600" src="https://github.com/user-attachments/assets/d32ec936-d238-4c92-9e1a-af6788d74cdf" alt="data grid row spanning" />
```
Comment (oliviertassinari) [inline CHANGELOG.md:14]: Adding a height would improve the UX.

```suggestion
  <img width="600" height="326" src="https://github.com/user-attachments/assets/d32ec936-d238-4c92-9e1a-af6788d74cdf" alt="data grid row spanning" />
```

so the content doesn't move while reading the changelog after the video loads.

---

On the video itself, I think that correct scale + x2 pixel density + higher fps would improve the UX. See the guide in https://www.notion.so/mui-org/High-quality-media-5a1686758f4f4267989b67a1c2c270da?pvs=4#03d3a56e4f3e4890be21a6d56a005194. Here is how it feel for comparison:

before

after

Comment (MBilalShafi) [reply CHANGELOG.md:14]: Cool, thanks for linking the page, it definitely looks better.

**Pull request merged**